cmd/csi-snapshotter/main.go

```diff
@@ -90,8 +90,9 @@ var (
 
 func main() {
 	klog.InitFlags(nil)
-	flag.Set("logtostderr", "true")
-	flag.Parse()
+	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
```

Comment:
Unless you want to cause a major version bump, the fix has to be the other way around: copy the parameter defined with pflag into the flag.CommandLine.